apps/http-server/packages/generic/src/lib.ts

```diff
@@ -23,3 +23,9 @@ export function valueOrFirst(text: string | string[] | undefined): string | unde
 		return text
 	}
 }
+
+/**
+ * The version of the package.json file
+ * Note: This variable is updated at build-time by scripts/prebuild.js
+ */
+export const PACKAGE_JSON_VERSION = '1.50.6'
```

Comment:
Would it make sense to put this into a separate file that's fully generated and gitignored to avoid potential conflicts?